src/unstract/sdk/index.py

```diff
@@ -357,6 +373,45 @@ def generate_file_id(
         hashed_index_key = ToolUtils.hash_str(json.dumps(index_key, sort_keys=True))
         return hashed_index_key
 
+    @deprecated("Deprecated class and method. Use Index and index() instead")
+    def index_file(
+        self,
+        tool_id: str,
+        embedding_type: str,
+        vector_db: str,
+        x2text_adapter: str,
+        file_path: str,
+        chunk_size: int,
+        chunk_overlap: int,
+        reindex: bool = False,
+        file_hash: Optional[str] = None,
+        output_file_path: Optional[str] = None,
+    ) -> str:
+        return self.index(
+            tool_id=tool_id,
+            embedding_instance_id=embedding_type,
+            vector_db_instance_id=vector_db,
+            x2text_instance_id=x2text_adapter,
+            file_path=file_path,
+            chunk_size=chunk_size,
+            chunk_overlap=chunk_overlap,
+            reindex=reindex,
+            file_hash=file_hash,
+            output_file_path=output_file_path,
+        )
+
+    @deprecated(
+        "Deprecated class and method. Use Index and query_texy_from_index() instead"
```

Comment:
@gaya3-zipstack `deprecated` library will throw warning that it is deprecated, we do not need to mention again.

In that case let's change message to `Use query_index() method of Index instance instead.`.

Reply:
Done